news/2025/2025-03-23-osulazer-updates-march-21-2025.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu!(lazer) Updates: March 21, 2025"
+date: 2025-03-23 18:30:00 +0000
+---
+
+A new osu!(lazer) update is delivered by a soft spring breeze.
+
+![](/wiki/shared/news/2025-03-23-osulazer-updates-march-21-2025/banner.jpg)
+
+Mother Nature is slowly rolling out the new Spring 2025 update, featuring warmer weather, more sun, and more greenery all across the board! Despite all of graphical and weather updates, the osu! developers still made time to focus and drop an update of their own for the osu!community.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cL4iwng19Ow" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### Editor
+
+- **The "discard unsaved changes" operation has been added to the editor.** Sometimes those new changes you've made just don't work out.
+- **The Kiai/star sound effect has now been added and made skinnable.** The new sound effect can be heard on the osu!argon skins, and you can use `fountain-shoot.wav` to place a sound of your own!
```

Comment:
it's a new addition since the last video

```suggestion
- **A new skinnable Kiai/star fountain sound effect has been added.** The new sound effect can be heard on the osu! argon skins, and you can use `fountain-shoot.wav` to place a sound of your own!
```